src/controllers/strings.js

```diff
@@ -0,0 +1,46 @@
+const {
+  sayHello,
+  uppercase,
+  lowercase,
+  firstCharacter,
+  firstCharacters,
+  countCharacters,
+} = require('../lib/strings');
+
+const practiceHello = (req, res) => {
+  res.status(200).send({ result: 'Hello, world!' });
+};
+
+const sayingHello = (req, res) => {
+  res.status(200).send({ result: sayHello(req.params.string) });
+};
+
+const toUpperCase = (req, res) => {
+  res.status(200).send({ result: uppercase(req.params.string) });
+};
+
+const toLowerCase = (req, res) => {
+  res.status(200).send({ result: lowercase(req.params.string) });
+};
+
+const getFirstCharacter = (req, res) => {
+  res.status(200).send({ result: firstCharacter(req.params.string) });
+};
+
+const countingCharacters = (req, res) => {
```

Comment:
There wasn't/isn't a test for this controller. Good to see extra work being done but try to follow the TDD cycle.